LircClient.py

```diff
@@ -83,32 +84,157 @@ class ClientInstantiationError(Exception):
     pass
 
 
+class Result(enum.Enum):
```

Comment:
Where does the "existing ReplyParser" come from? Author? Copyright? If it is "existing", possibly it can/should be deployed in another way?